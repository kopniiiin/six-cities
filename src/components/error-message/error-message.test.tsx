import * as React from "react";
import {configure, shallow} from "enzyme";
import * as Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";

import ErrorMessage from "./error-message";

configure({adapter: new Adapter()});

const testMocks = {text: `error`};

describe(`snapshot test: ErrorMessage component`, () => {
  it(`should render correctly`, () => expect(
      toJson(shallow(<ErrorMessage {...testMocks}/>))
  ).toMatchSnapshot());
});
