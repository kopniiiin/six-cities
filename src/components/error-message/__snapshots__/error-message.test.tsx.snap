// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test: ErrorMessage component should render correctly 1`] = `
<div
  className="error-message"
>
  error
</div>
`;
